tests/Microsoft.Bot.Builder.Tests/BotStateSetTests.cs

```diff
@@ -26,12 +24,12 @@ public void BotStateSet_Properties()
 
             var stateSet = new BotStateSet(userState, convState);
 
-            Assert.AreEqual(stateSet.BotStates.Count, 2);
-            Assert.IsNotNull(stateSet.BotStates.OfType<UserState>().First());
-            Assert.IsNotNull(stateSet.BotStates.OfType<ConversationState>().First());
+            Assert.Equal(stateSet.BotStates.Count, 2);
```

Comment:
>tateSet.BotStates.Count, 2) [](start = 26, length = 27)

The expected value (2) should be the first parameter: Assert.Equal(2, stateSet.BotStates.Count);

Reply:
updated.